test/acceptance/keywords/cookies.robot

```diff
@@ -48,9 +48,14 @@ Get Cookies When There Are None
     ${cookies}=    Get Cookies
     Should Be Equal    ${cookies}    ${EMPTY}
 
+Get Cookie Expiry Set By Selenium
```

Comment:
FF indeed seems to have a problem with cookies, please add a tag to the test `Known_Issue_Firefox` and CI will automatically mark this test as noncritical for Firefox.

Did you happen to try with other browsers, like IE, Edge or Safari, that do they suffer from the same problem?